layers/+lang/rust/README.org

```diff
@@ -33,6 +34,22 @@ To enable auto-completion, ensure that the =auto-completion= layer is enabled.
 [[http://doc.crates.io/index.html][Cargo]] is a project management command line tool for Rust. Installation
 instructions can be found on the main page of [[http://doc.crates.io/index.html][Cargo]].
 
+** Rustfmt
+
+To formatting Rust code according to style guidelines using [[https://github.com/rust-lang-nursery/rustfmt][rustfmt]].
```

Comment:
I think it should read:
> Format Rust code ...

(no need to close this PR, just force-push to this branch)